lib/kubeclient/common.rb

```diff
@@ -67,8 +67,8 @@ def rest_client
             ssl_client_cert: @ssl_options[:client_cert],
             ssl_client_key: @ssl_options[:client_key]
           }
-          endpoint_with_version = @api_endpoint.merge(@api_endpoint.path + '/' \
-                                                      + @api_version)
+          endpoint_with_version = @api_endpoint.merge(@api_endpoint.path + '/' +
+                                                      @api_version)
```

Comment:
I think we preferred `"#{@api_endpoint.path}/#{@api_version}"` in other parts.

Same below as well.